Makefile

```diff
@@ -111,30 +116,39 @@ generate-%: ${BUILD_FOLDER}
 ##################
 
 .PHONY: build docs
-build: lint unit-test build-minimal
+
+validate: lint unit-test
+
+build: validate build-minimal
 
 build-all: build-in-docker
 
 build-in-docker:
 	skipper make build-image operator-bundle-build
 
 build-minimal: $(BUILD_FOLDER)
-	CGO_ENABLED=0 go build -o $(BUILD_FOLDER)/assisted-service cmd/main.go
-	CGO_ENABLED=0 go build -o $(BUILD_FOLDER)/assisted-service-operator cmd/operator/main.go
+	CGO_ENABLED=0 go build $(DEBUG_ARGS) -o $(BUILD_FOLDER)/assisted-service cmd/main.go | CGO_ENABLED=0 go build $(DEBUG_ARGS) -o $(BUILD_FOLDER)/assisted-service-operator cmd/operator/main.go
```

Comment:
Why `|`?

Reply:
Also whatever you end up putting there, please keep them on separate lines with `\`:

e.g.
```bash
CGO_ENABLED=0 go build $(DEBUG_ARGS) -o $(BUILD_FOLDER)/assisted-service cmd/main.go | \
CGO_ENABLED=0 go build $(DEBUG_ARGS) -o $(BUILD_FOLDER)/assisted-service-operator cmd/operator/main.go
```